neuro_san/internals/run_context/langchain/llms/default_llm_factory.py

```diff
@@ -215,7 +263,7 @@ def create_full_llm_config(self, config: Dict[str, Any]) -> Dict[str, Any]:
 
         return full_config
 
-    def get_chat_class_args(self, chat_class_name: str, use_model_name: str) -> Dict[str, Any]:
+    def get_chat_class_args(self, chat_class_name: str, use_model_name: Optional[str] = None) -> Dict[str, Any]:
```

Comment:
FWIW: We haven't really been using Optional in our typing unless its completely necessary.
The default arg of None communicates that pretty clearly.

I think the only time we would really need Optional[] is if we were sending this through something very rigorous like Pydantic and/or deriving from an interface that specifically had Optional as part of it.